docs/Developers/Development_Setup.rst

```diff
@@ -52,6 +52,10 @@ Using virtualenv
         $ virtualenv coala-venv
 
 NOTE:
+Depending on your system,you most likely need :code:`sudo` to install virtualenv
```

Comment:
Space after comma and period at the end. And please add double `-s around the sudo